sechub-web-ui/src/components/ProjectsList.vue

```diff
@@ -40,18 +41,24 @@
 
 <script>
   import defaultClient from '@/services/defaultClient'
+  import { useRouter } from 'vue-router'
+  import { ref } from 'vue'
+  import { useProjectStore } from '@/stores/projectStore'
 
   export default {
-    name: 'ProjectsComponent',
+    name: 'ProjectListComponent',
 
     setup () {
       const projects = ref([])
       const loading = ref(true)
       const error = ref(null)
+      const router = useRouter()
+      const store = useProjectStore()
 
       onMounted(async () => {
         try {
           projects.value = await defaultClient.withProjectApi.getAssignedProjectDataList()
+          store.storeProjects(projects)
```

Comment:
currently you only store the projects - but it seems you do not check if already loaded and use the store as a cache.

Maybe this will done later - at the moment I did not find anything that uses `getProjectById` (at least directly) to obtain data from store.

Reply:
You observed this correctly - the store is intended for the next issue, where it will be used to show the project details. At the moment, the functionality to check if a project is already loaded and use the store as a cache is not implemented - we could address this Issue in upcoming tasks.